src/Integration/Telemetry/Payload/ServerNotifications.cs

```diff
@@ -28,16 +31,16 @@ public sealed class ServerNotifications
         public bool IsDisabled { get; set; }
 
         [JsonProperty("count_by_type")]
-        public ServerNotificationCounters ServerNotificationCounters { get; set; } = new ServerNotificationCounters();
-    }
-
-    public sealed class ServerNotificationCounters
-    {
-        [JsonProperty("QUALITY_GATE")]
-        public ServerNotificationCounter QualityGateNotificationCounter { get; set; } = new ServerNotificationCounter();
+        [XmlIgnore]
+        public Dictionary<string, ServerNotificationCounter>  ServerNotificationCounters { get; set; } = new Dictionary<string, ServerNotificationCounter>();
 
-        [JsonProperty("NEW_ISSUES")]
-        public ServerNotificationCounter NewIssuesNotificationCounter { get; set; } = new ServerNotificationCounter();
+        [JsonIgnore]
+        [XmlArray(nameof(ServerNotificationCounters))]
```

Comment:
A dictionary is not xml serializable, this is a workaround (based on https://stackoverflow.com/a/34445868/339720